src/index.tsx

```diff
@@ -862,17 +861,17 @@ class DraggableFlatList<T> extends React.Component<
             ? styles.hoverComponentHorizontal
             : styles.hoverComponentVertical,
           {
-            opacity: this.hoverComponentOpacity,
+            opacity: this.hoverComponentOpacity[" __value"],
```

Comment:
why is this necessary?

Reply:
It turns out it's not necessary. Reverted it, thank you!